examples/rcn/load_data.py

```diff
@@ -0,0 +1,78 @@
+# export
```

Comment:
If we use `tensorflow_datasets` to load MNIST digits, I see no reason for this to be a separate file. We should be able to do data loading and preprocessing in a few lines in the **STANDALONE** notebook